apps/workers/openaiWorker.ts

```diff
@@ -88,6 +94,14 @@ You must respond in JSON with the key "tags" and the value is list of tags.
 CONTENT START HERE:
 `;
 
+const PDF_PROMPT_BASE = `
```

Comment:
The difference between this and the text prompt is not that big. I think we can just continue using the text prompt even for pdfs, what do you think?

Reply:
You are correct, I'm a big fan of generalization when possible. 
I'll combine them.